pdbx/version.py

```diff
@@ -0,0 +1,6 @@
+"""PDBX Version number."""
+# Store the version here so:
+# 1) we don't load dependencies by storing it in __init__.py
+# 2) we can import it in setup.py for the same reason
+# 3) we can import it into your module module
+__version__ = "2.0.3"
```

Comment:
Do you now have to manually update the version number?

Reply:
Yes.  I've also just updated the code to make this module more "private".